toolchains/sysimage/build_ext4_image.py

```diff
@@ -221,6 +218,8 @@ def main():
         "hash_seed=c61251eb-100b-48fe-b089-57dea7368612",
         "-U",
         "clear",
+        "-d",
+        os.path.join(fs_basedir, limit_prefix),
```

Comment:
Same as https://github.com/dfinity/ic/pull/3476#issuecomment-2599430289, we should make sure to investigate just in case.

Reply:
https://github.com/dfinity/ic/pull/3476#issuecomment-2608074971